.gitignore

```diff
@@ -17,3 +17,6 @@ ci-scripts/
 *.tar.gz
 
 /provider/cmd/pulumi-resource-*/schema.go
```

Comment:
Looks like I copy/pasted this file from the Go boilerplate. I don't think this line is relevant for this repo. Mind deleting it while making changes here?

Reply:
Done!